tests/model_inheritance/tests.py

```diff
@@ -331,6 +331,13 @@ def test_update_works_on_parent_and_child_models_at_once(self):
         self.assertFalse(r1.serves_hot_dogs)
         self.assertEqual(r1.name, "Demon Puppies")
 
+    def test_update_query_counts_for_proxy(self):
+        """
+        #28820
```

Comment:
Docstrings should state the expected behavior. Ticket references should be reserved for obscure issues. These guidelines are from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).